src/i18n/_locales/en_US/messages.json

```diff
@@ -246,6 +246,23 @@
   "i18n_font_size" : {
     "message" : "FONT SIZE"
   },
+  "i18n_font_selection" : {
+    "message" : "FONT FACE"
+  },
+  "i18n_fonttype-default" : {
+    "message" : "Default"
+  },
+    "i18n_fonttype-odf" : {
+    "message" : "Open Dyslexic Font"
+  },
```

Comment:
TODO: this should not be hard-coded, instead the message string should be picked-up from `readium_js_viewer/ModuleConfig.fonts[n].displayName`